client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.html

```diff
@@ -2,7 +2,7 @@
     <div class="title">
         <h1>Active Wildfires Map</h1>
         <h2 class="wildfire-count">
-            <wf-icon iconName="local_fire_department" iconSize="small"></wf-icon>
+            <wf-icon class="fire-icon" iconName="local_fire_department" iconSize="small"></wf-icon>
```

Comment:
Is this the right icon? It looks different than the one screenshotted in the WFNEWS-397 ticket. I had tried the span class of "material-icons-outlined" when looking at this ticket before and it had been showing up as plain English however.

Reply:
It's the right name.... but it definitely looks different.

I want to look into the icon library we're using and see if maybe it's out of date with the latest/greatest from google